tools/c7n_azure/c7n_azure/function.py

```diff
@@ -25,7 +25,7 @@
 
 try:
     import azure.functions as func
-    from azure.functions.queue import QueueMessage
+    from azure.functions import QueueMessage
```

Comment:
this should not be in a naked except IMportError: pass

the except clause should re raise if not on py 2.7 (check sys.version_info)

Reply:
This isn't required for custodian CLI users, this code only runs in Azure, but I guess now that it is in our setup.py they should have it anyway.